Tests/conf.json

```diff
@@ -5597,6 +5597,24 @@
             "integrations": "Netcraft V2",
             "playbookID": "Test - Netcraft",
             "timeout": 2400
+        },
+        {
```

Comment:
I think something went wrong here - please check that those test PBs already exist with an instance. If not, we need to check this with the code owner of the tests.